uniflow/flow/transform/model.py

```diff
@@ -17,101 +17,9 @@
 MAX_ATTEMPTS = 3
 
 
-class Model:
+class LLMDataProcessorJson(LLMDataProcessor):
     """
-    Model Class provides general functions for various LLMs, including
-    serialize the input data, apply a given LLM on the serialized data,
-    and deserialize the model outputs.
-    """
-
-    def __init__(
-        self,
-        guided_prompt_template: GuidedPrompt,
-        model_config: Dict[str, Any],
-    ) -> None:
-        """Initialize Model class.
-
-        Args:
-            guided_prompt_template (GuidedPrompt): Guided prompt template.
-            model_config (Dict[str, Any]): Model config.
-        """
-        model_server_cls = ModelServerFactory.get(model_config["model_server"])
-        self._model_server = model_server_cls(model_config)
-        self._guided_prompt_template = guided_prompt_template
-        self._prompt_keys = []
-
-    def _serialize(self, data: List[Context]) -> List[str]:
-        """Serialize data.
-
-        Args:
-            data (List[Context]): Data to serialize.
-
```

Comment:
qq: why all this code is deleted?

Reply:
model and json model have completed different serialize and de-serialize code. why delete the code here?

Reply:
It's not deleted, but renamed as `LLMDataProcessor` and moved to `uniflow/op/transform/model_op.py`